package.json

```diff
@@ -78,6 +79,7 @@
         "command-exists": "~1.2.9",
         "compare-versions": "~3.6.0",
         "dayjs": "~1.10.8",
+        "esm": "^3.2.25",
```

Comment:
Is this needed? This package hasn't been updated since 2019

Reply:
According to recent comments, that project was abandoned.

Reply:
Yes unfortunately this is required by the axios-cached-dns-resolve lib. It's required by knex so is already installed with Uptime Kuma.